sycl/plugins/cuda/CMakeLists.txt

```diff
@@ -6,7 +6,8 @@ message(STATUS "Including the PI API CUDA backend.")
 
 find_package(CUDA 10.0 REQUIRED)
 
-add_library(cudadrv SHARED IMPORTED)
+# Make imported library global to use it within the project.
+add_library(cudadrv SHARED IMPORTED GLOBAL)
```

Comment:
cudadrv is found by find_package(CUDA...) ?

Reply:
The [FindCUDA package](https://cmake.org/cmake/help/v3.17/module/FindCUDA.html) (deprecated since CMake 3.10) does not find or export the CUDA driver.

The new [FindCUDAToolkit](https://cmake.org/cmake/help/v3.17/module/FindCUDAToolkit.html) (avialable from CMake 3.17 on) export the CUDA driver library.